run_server.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
+set -ex
```

Comment:
is this an intended change?

Reply:
not intended - it was from an earlier commit